values.yaml

```diff
@@ -458,9 +458,6 @@ scheduler:
   # scheduler.env -- Redash scheduler specific environment variables.
   env: {}
 
-  # scheduler.replicaCount -- Number of scheduler pods to run
```

Comment:
Will need to update README and perhaps add a changelog note, but otherwise this looks good - thanks!

Reply:
I've updated README, but I'm not sure about a version number in the changelog note.